### PR TITLE
chore(zh-tw): run prettier format on web/ (except api/js)

### PR DESCRIPTION
**Pull request by yin1999**
### Description

run prettier format on web/ (except api/js)

### Related issues and pull requests

Part of mdn/mdn#362

### CODE REVIEW

**Comment by queengooborg** [comment]
Given how many files are touched in this PR, could we split it into two or three PRs?

Also, it appears that there's some content synchronization in this PR as well -- I don't really feel qualified to review translation work, would it be alright to cherry-pick those changes for others to review?

**Comment by fwqaaq** [comment]
> Given how many files are touched in this PR, could we split it into two or three PRs?
> 
> Also, it appears that there's some content synchronization in this PR as well -- I don't really feel qualified to review translation work, would it be alright to cherry-pick those changes for others to review?

There's no translated content, only some formatting has been changed. You can review them with confident.

**Comment by queengooborg** [comment]
> There's no translated content, only some formatting has been changed. You can review them with confident.

There are non-formatting changes, though.  :)

File: files/zh-tw/web/css/_colon_lang/index.md [hunk 1]
```diff
@@ -1,26 +1,23 @@
...
 ## 語法 Syntax
 
 ### 語法格式
 
-{{csssyntax}}
+```css-nolint
+:lang(<language-code>) {
+  /* ... */
+}
+```
```
Comment (queengooborg) [inline files/zh-tw/web/css/_colon_lang/index.md:20]: (Example of non-Prettier, content sync changes -- this one's a very small change that I feel confident to review, but there are others I'm not quite confident in reviewing!)

Comment (yin1999) [reply files/zh-tw/web/css/_colon_lang/index.md:20]: Thanks for the note. I'll split the content sync changes to other PRs. I just forgot to list the pending review comments (sorry about this, but I still like to split them, not list those comments here)

File: files/zh-tw/web/css/_colon_target/index.md [hunk 2]
```diff
@@ -1,43 +1,45 @@
...
 
 {{CSSRef}}
 
-The` :target `[pseudo-class](/zh-TW/docs/Web/CSS/Pseudo-classes) represents the unique element, if any, with an **id** matching the fragment identifier of the URI of the document..
+The`:target`[pseudo-class](/zh-TW/docs/Web/CSS/Pseudo-classes) represents the unique element, if any, with an **id** matching the fragment identifier of the URI of the document..
 
 URIs with fragment identifiers link to a certain element within the document, known as the _target element_. For instance, here is a URI pointing to an _anchor_ named section2:
 `http://example.com/folder/document.html#section2`
-The _anchor_ can be any element with an` id `attribute, e.g. `<h1 id="section2">` in our example. The _target element_ `h1` can be represented by the `:target` pseudo-class.
+The _anchor_ can be any element with an`id`attribute, e.g. `<h1 id="section2">` in our example. The _target element_ `h1` can be represented by the `:target` pseudo-class.
```
Comment (queengooborg) [inline files/zh-tw/web/css/_colon_target/index.md:12]:
```suggestion
The _anchor_ can be any element with an `id` attribute, e.g. `<h1 id="section2">` in our example. The _target element_ `h1` can be represented by the `:target` pseudo-class.
```

**Review by queengooborg — APPROVED** [review]

**Pull request merged**